### ISSUE TITLE
services.AddSwaggerGen(c => {}) not working

### ISSUE THREAD

**Issue by SaschaHerrmann** [domaindrivendev/Ahoy#60]
Hi,

I am using ASP.Net 5 Swagger for my API, works very good. For Authentication i want to use Oauth2 and tried to replicate the example:

`services.AddSwaggerGen(c =>
            {
                c.AddSecurityDefinition("oauth2", new OAuth2Scheme
                {
                    Type = "oauth2",
                    Flow = "implicit",
                    AuthorizationUrl = "http://petstore.swagger.io/api/oauth/dialog",
                    Scopes = new Dictionary<string, string>
                        {
                            { "read", "read access" },
                            { "write", "write access" }
                        }
                });
                c.OperationFilter<AssignSecurityRequirements>();
            });`

My project.json:

`"Swashbuckle.SwaggerGen": "6.0.0-rc1-final",
    "Swashbuckle.SwaggerUi": "6.0.0-rc1-final"`

The example is working, but in my project the extension for `services.AddSwaggerGen` does not accept any argument. The using statement 

`using Swashbuckle.SwaggerGen.Generator;` 
The error is that SwaggerGen does not contain the namespace Generator.

Anything broken in the nuget package?

**Comment by kudoz83** [comment]
Confirming this issue in 6.0.0-rc1-final from nuget.

**Comment by kudoz83** [comment]
The following appears to do it...

````
services.ConfigureSwaggerDocument(options =>
			{
				options.SingleApiVersion(new Info
				{
					Version = "v1",
					Title = "<TITLE>",
					Description = "<DESCRIPTION>"
				});

				options.SecurityDefinitions.Add("oauth2", new OAuth2Scheme {
					Type = "oauth2",
					Flow = "implicit",
					AuthorizationUrl = "<URL>",
					Scopes = new Dictionary<string, string> { { "full", "full control" } }
				});

				options.OperationFilter<AssignSecurityRequirements>();
			});
			
			services.ConfigureSwaggerSchema(options => {
				options.DescribeAllEnumsAsStrings = true;
			});
````

**Comment by patrickmichalina** [comment]
Yea, this is broken :(

**Comment by domaindrivendev** [comment]
Should be fixed in the 6.0.0-beta9 release.

**Comment by AriaAkbariyeh** [comment]
not working !
